tokio/src/runtime/task/mod.rs

```diff
@@ -193,6 +193,9 @@ use std::{fmt, mem};
 ///   task completes, the same ID may be used for another task.
 /// - Task IDs are *not* sequential, and do not indicate the order in which
 ///   tasks are spawned, what runtime a task is spawned on, or any other data.
+/// - Task ID of a task can be obtained via the functions
+///   [`task::try_id()`](crate::runtime::task::try_id()) and
+///   [`task::id()`](crate::runtime::task::id()).
```

Comment:
the way this is worded, it sounds kind of like these functions can be used to get the task ID of _any_ task...what if we re-worded this to:
```suggestion
/// - The task ID of the currently running task can be obtained via the
///   [`task::try_id()`](crate::runtime::task::try_id()) and
///   [`task::id()`](crate::runtime::task::id()) functions.
```

and it might also be worth mentioning that `JoinHandle::id` can also be used to access a task's ID through a `JoinHandle`...

Reply:
also, do we know if these links actually work correctly in the RustDoc documentation? the `tokio::runtime::task` module is not public, and these functions are re-exported publicly from `tokio::task`, so should the link references instead refer to `crate::task::id()` etc?

Reply:
I've changed them to `crate::task::id`. Right now when I build docs, I get some unresolved link errors, so I can't test them right now, but I will test them once I get the doc building working.